spaceship/lib/spaceship/tunes/app_submission.rb

```diff
@@ -132,6 +131,13 @@ def complete!
         end
         raw_data_clone.delete("version")
 
+        if !self.add_id_info_uses_idfa.nil? && self.add_id_info_uses_idfa == true
+          raw_data_clone.set(
```

Comment:
Any chance you could just add a comment in here describing what this is doing? Its just a bit unclear as it is 😄

Reply:
There you go 😄